src/routes/settings.tsx

```
@@ -180,6 +192,27 @@ function SettingsPage() {
                     />
                 </div>
 
+                <div className="form-control w-full">
+                    <label className="label">
+                        <span className="label-text">Ping Delay (ms)</span>
+                        <span className="label-text-alt font-mono">{pingDelay}ms</span>
+                    </label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)